### DISCUSSION TITLE
Is codeformatter.exe still used?

### DISCUSSION THREAD

**Discussion by RichardMelito** [dotnet/runtime#65099] [Q&A]
On the coding style page it states that the .NET Codeformatter Tool is used during builds. It links to the [codeformatter.exe repo](https://github.com/dotnet/codeformatter). However, that project seems out of date and I couldn't find any other references to it in the runtime repo. Is the [dotnet-format](https://github.com/dotnet/format) tool being used instead? If so, then the coding style page should be updated accordingly.

**Comment by danmoseley** [comment]
I think that's right .. @stephentoub  do you know?

**Reply by RichardMelito** [reply]
Which is right: using codeformatter or using dotnet-format?

**Reply by danmoseley** [reply]
@RichardMelito did you try running dotnet-format on part of this repo? I'm curious how well it works -- if well, then perhaps that answers itself. But I do not know that we run formatting much except when bringing in a chunk of old code. We get warnings in Visual Studio generated by the .editorconfig and address those.

**Reply by RichardMelito** [reply]
@danmoseley I did not try it myself. I just noticed that dotnet-format is much newer than codeformatter and is part of the SDK, so it seems logical that dotnet-format would replace it. 

Separately, I tried using codeformatter on a private repo and I could not get it do anything at all. Meanwhile, dotnet-format worked pretty easily.

Edit: I also found no references to codeformatter in the repo (except for a mention in the style guide) but found a few references to dotnet-format.

**Reply by danmoseley** [reply]
Gotcha. Suffice it to say I don't think we use it much in recent years (as code porting effort to .NET Core is behind us), so if you or someone can establish that dotnet format behaves reasonably well on the code style here, we'd probably be glad to have a PR to fix the docs.

**Reply by stephentoub** [reply]
To my knowledge we no longer use codeformatter.
